docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -224,9 +231,17 @@ spec:
         service:
           metadata:
             annotations:
+ifeval::["{OpenStackPreviousInstaller}" == "director"]
```

Comment:
There is also "TripleO" value which is used for upstream rendering: https://github.com/openstack-k8s-operators/data-plane-adoption/blob/d7fd97eeefdd7b7397e69c19fcc75497000ba423/docs_user/adoption-attributes.adoc#L19 .

Since we're unlikely to have a third one, i would suggest changing all these lines:

```
ifeval::["{OpenStackPreviousInstaller}" == "director"]
```

into this:

```
ifeval::["{OpenStackPreviousInstaller}" != "director_operator"]
```

Reply:
ok changing